assets/component-mega-menu.css

```diff
@@ -1,38 +1,68 @@
 .mega-menu {
-  position: initial;
+  position: static;
 }
 
-.mega-menu-content {
+.mega-menu__content {
   background-color: rgb(var(--color-background));
   border-left: 0;
   border-radius: 0;
   border-right: 0;  
   left: 0;
-  padding-bottom: 3.2rem;
-  padding-top: 3.2rem;
+  max-height: calc(var(--viewport-height, 100vh) - var(--header-bottom-position, 200px) - 4rem);
```

Comment:
Is there a benefit to using `px` as the default vs. `rem` in the default value case for the css variable?

Reply:
Updated to `rem` https://github.com/Shopify/dawn/pull/1338/commits/630713d800865d859b3024968a69c49e295a503f